docs/cloud/provision/azure/considerations.md

```diff
@@ -1,31 +1,30 @@
 # Considerations for Microsoft Azure
 
-Due to differences between Microsoft Azure and other cloud providers, the provisioning process in Event Store Cloud is different from AWS and GCP. Event Store Cloud also has certain limitations when it comes to Azure, which you can also find on this page.
+Due to differences between Microsoft Azure and other cloud providers, the provisioning process in Event Store Cloud is different from AWS and GCP. We’ve made a list of these differences here in order to help you make an informed decision about Cloud providers.
 
 ## Network peering
 
-When creating peering links in Azure require the user to configure a security principle referencing the application ID of Event Store Cloud, as well as configure and apply a role allowing that principle to modify the network resource of the remote network. Event Store Cloud console provides the necessary Azure CLI commands at creation time, to support this operation.
+When creating peering links Azure require the user to configure a security principle referencing the application ID of Event Store Cloud, and also configure and apply a role allowing that principle to modify the network resource of the remote network. Event Store Cloud console provides the necessary Azure CLI commands at creation time, to support this operation.
```

Comment:
Shouldn't `security principle` be `security principal`
`principal` is the object created in Azure AD
@oskardudycz , @mat-mcloughlin

Reply:
@dairyleia Yves is right this should be principal

Reply:
@oskardudycz how do I edit this one?

Reply:
The entire thing, which I wrote, is a mess :)

> When creating peering links Azure require the user to configure a security principle referencing the application ID of Event Store Cloud, and also configure and apply a role allowing that principle to modify the network resource of the remote network.

Should be:

When creating a peering link, Azure requires the user to configure a security _principal_, referencing the application ID of Event Store Cloud, and also configure and apply a role allowing that principal to modify the network resource of the remote network.

Reply:
Alternative:

When creating a peering link in Azure, the user needs to configure a security principal, referencing the application ID of Event Store Cloud, and also configure and apply a role allowing that principal to modify the network resource of the remote network.

Reply:
Also, replace "principle" to "principal" everywhere.